To: vim_dev@googlegroups.com
Subject: Patch 7.3.423
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.423 
Problem:    Small mistakes in comments, proto and indent. 
Solution:   Fix the mistakes. 
Files:      src/ex_cmds2.c, src/structs.h, src/ui.c, src/proto/ex_docmd.pro 


*** ../vim-7.3.422/src/ex_cmds2.c	2011-10-20 21:38:37.000000000 +0200
--- src/ex_cmds2.c	2011-11-30 11:02:20.000000000 +0100
***************
*** 1939,1945 ****
  }
  
  /*
!  * Return TRUE if window "win" is editing then file at the current argument
   * index.
   */
      static int
--- 1939,1945 ----
  }
  
  /*
!  * Return TRUE if window "win" is editing the file at the current argument
   * index.
   */
      static int
*** ../vim-7.3.422/src/structs.h	2011-05-19 13:40:47.000000000 +0200
--- src/structs.h	2011-09-10 13:48:53.000000000 +0200
***************
*** 1846,1852 ****
  				       often, keep it the first item!) */
  
  #if defined(FEAT_SYN_HL) || defined(FEAT_SPELL)
!     synblock_T	*w_s;
  #endif
  
  #ifdef FEAT_WINDOWS
--- 1846,1852 ----
  				       often, keep it the first item!) */
  
  #if defined(FEAT_SYN_HL) || defined(FEAT_SPELL)
!     synblock_T	*w_s;		    /* for :ownsyntax */
  #endif
  
  #ifdef FEAT_WINDOWS
*** ../vim-7.3.422/src/ui.c	2011-07-07 16:20:45.000000000 +0200
--- src/ui.c	2012-01-21 14:03:42.000000000 +0100
***************
*** 2363,2376 ****
  	       XtLastTimestampProcessed(XtDisplay(myShell)),
  	       clip_x11_convert_selection_cb, clip_x11_lose_ownership_cb,
  	       NULL) == False)
! 	return FAIL;
      }
      else
  #endif
      {
  	if (!XChangeProperty(XtDisplay(myShell), XtWindow(myShell),
  		  cbd->sel_atom, timestamp_atom, 32, PropModeAppend, NULL, 0))
! 	return FAIL;
      }
      /* Flush is required in a terminal as nothing else is doing it. */
      XFlush(XtDisplay(myShell));
--- 2363,2376 ----
  	       XtLastTimestampProcessed(XtDisplay(myShell)),
  	       clip_x11_convert_selection_cb, clip_x11_lose_ownership_cb,
  	       NULL) == False)
! 	    return FAIL;
      }
      else
  #endif
      {
  	if (!XChangeProperty(XtDisplay(myShell), XtWindow(myShell),
  		  cbd->sel_atom, timestamp_atom, 32, PropModeAppend, NULL, 0))
! 	    return FAIL;
      }
      /* Flush is required in a terminal as nothing else is doing it. */
      XFlush(XtDisplay(myShell));
*** ../vim-7.3.422/src/proto/ex_docmd.pro	2010-08-15 21:57:28.000000000 +0200
--- src/proto/ex_docmd.pro	2011-01-16 00:22:21.000000000 +0100
***************
*** 1,7 ****
  /* ex_docmd.c */
  void do_exmode __ARGS((int improved));
  int do_cmdline_cmd __ARGS((char_u *cmd));
! int do_cmdline __ARGS((char_u *cmdline, char_u *(*getline)(int, void *, int), void *cookie, int flags));
  int getline_equal __ARGS((char_u *(*fgetline)(int, void *, int), void *cookie, char_u *(*func)(int, void *, int)));
  void *getline_cookie __ARGS((char_u *(*fgetline)(int, void *, int), void *cookie));
  int checkforcmd __ARGS((char_u **pp, char *cmd, int len));
--- 1,7 ----
  /* ex_docmd.c */
  void do_exmode __ARGS((int improved));
  int do_cmdline_cmd __ARGS((char_u *cmd));
! int do_cmdline __ARGS((char_u *cmdline, char_u *(*fgetline)(int, void *, int), void *cookie, int flags));
  int getline_equal __ARGS((char_u *(*fgetline)(int, void *, int), void *cookie, char_u *(*func)(int, void *, int)));
  void *getline_cookie __ARGS((char_u *(*fgetline)(int, void *, int), void *cookie));
  int checkforcmd __ARGS((char_u **pp, char *cmd, int len));
*** ../vim-7.3.422/src/version.c	2012-02-04 20:17:21.000000000 +0100
--- src/version.c	2012-02-04 21:56:43.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     423,
  /**/

-- 
In many of the more relaxed civilizations on the Outer Eastern Rim of the
Galaxy, "The Hitchhiker's Guide to the Galaxy" has already supplanted the
great "Encyclopedia Galactica" as the standard repository of all knowledge
and wisdom, for though it has many omissions and contains much that is
apocryphal, or at least wildly inaccurate, it scores over the older, more
pedestrian work in two important respects.
First, it is slightly cheaper; and second, it has the words "DON'T PANIC"
inscribed in large friendly letters on its cover.
		-- Douglas Adams, "The Hitchhiker's Guide to the Galaxy"

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
